src/Packetery/Module/Order/Metabox.php

```diff
@@ -197,8 +189,19 @@ public function register(): void {
 		add_action(
 			'admin_init',
 			function () {
-				$this->order_form = $this->formFactory->create();
-				$this->add_fields();
+				$this->orderForm = $this->orderDetails->create();
```

Comment:
Na všech ostatních místech používáš $this->orderDetailsFormFactory->create() a zde $this->orderDetails->create(). Jaký je k tomu důvod ?

Reply:
Někde se mi to tam konfliktilo s jiný classovým parametrem, ale opravil jsem to.